### PR TITLE
Add purescript

### PR DESCRIPTION
**Pull request by matoruru**
I added PureScript config.
If it's OK, please merge this.

### CODE REVIEW

File: autoload/quickrun.vim [hunk 1]
```diff
@@ -528,6 +528,14 @@ let g:quickrun#default_config = {
...
 \   'exec': ['%c %o < %s'],
 \ },
+\ 'purescript': {
+\   'type': executable('pulp') ? 'purescript/pulp' : '',
+\ },
+\ 'purescript/pulp': {
+\   'command': 'pulp',
+\   'cmdopt': '--monochrome',
+\   'exec': '%c %o run',
+\ },
```
Comment (thinca) [inline autoload/quickrun.vim]: Please add the new entries in alphabetical order.

Comment (matoruru) [reply autoload/quickrun.vim]: OK, thank you for your advice.

**Comment by matoruru** [comment]
I've changed it as you requested.
Is it OK?

**Comment by thinca** [comment]
Could you squash?  (Sorry for many requests 🙇)

**Comment by matoruru** [comment]
Of course OK. It's my mistake, you don't have to apologize! :)

**Comment by matoruru** [comment]
I squashed those commits!

**Comment by thinca** [comment]
Thank you!

**Pull request merged**